db/R/dbfiles.R

```diff
@@ -34,16 +34,17 @@
 dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate, mimetype, formatname, parentid=NA, con, hostname=fqdn()) {
```

Comment:
not sure if posteriors was working in the first place since it doesn't have a `format_id` column. Should it?

```r
  posteriorid <- db.query(paste0("SELECT id FROM posteriors WHERE pft_id=", pftid, " AND format_id=", formatid), con)[['id']]
```